### PR TITLE
datree: init at 0.13.4

### PR DESCRIPTION
**Pull request by jceb**
###### Motivation for this change

Add `datree` packages.

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/#sec-conf-file))
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: maintainers/maintainer-list.nix [hunk 1]
```diff
@@ -12779,5 +12779,10 @@
     github = "mbprtpmnr";
     githubId = 88109321;
   };
-
+  jceb = {
```
Comment (SuperSandro2000) [inline maintainers/maintainer-list.nix:12782]: This should be already in master now.

File: pkgs/development/tools/datree/default.nix [hunk 2]
```diff
@@ -0,0 +1,31 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "datree";
+  # test
```
Comment (SuperSandro2000) [inline pkgs/development/tools/datree/default.nix:5]:
```suggestion
```

File: pkgs/development/tools/datree/default.nix [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+    sha256 = "0q399ydz6k3p88dlyk5qzki76wiiw3zvkfh40vh9y9wzala8dhhp";
+  };
+
+  vendorSha256 = "1l7k1ss0iqlkjldi0jqlpfrlzj1zkmbzi98m5p3d12rr84i7in2k";
+
+  ldflags = let t = "sigs.k8s.io/kustomize/api/provenance"; in
+    [
+      "-s"
+      "-X github.com/datreeio/datree/cmd.CliVersion=${version}"
+    ];
```
Comment (SuperSandro2000) [inline pkgs/development/tools/datree/default.nix:21]:
```suggestion
  ldflags = [
    "-s"
    "-X github.com/datreeio/datree/cmd.CliVersion=${version}"
  ];
```

File: pkgs/development/tools/datree/default.nix [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+  doCheck = false;
```
Comment (SuperSandro2000) [inline pkgs/development/tools/datree/default.nix:23]: Why did you disable them?

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]
@ofborg eval

File: maintainers/maintainer-list.nix (hunk 1 above)
Comment (jceb) [reply maintainers/maintainer-list.nix:12782]: I don't see it yet.

File: pkgs/development/tools/datree/default.nix (hunk 4 above)
Comment (jceb) [reply pkgs/development/tools/datree/default.nix]: No specific reason, I copied it from another package. I enabled them now.

File: maintainers/maintainer-list.nix (hunk 1 above)
Comment (SuperSandro2000) [reply maintainers/maintainer-list.nix:12782]: I thought I merged another PR with it. Apparently not but please sort the entry alphabetically.

File: pkgs/development/tools/datree/default.nix [hunk 5]
```diff
@@ -0,0 +1,29 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "datree";
+  version = "0.13.4";
+
+  src = fetchFromGitHub {
+    owner = "datreeio";
+    repo = "datree";
+    rev = "${version}";
```
Comment (SuperSandro2000) [inline pkgs/development/tools/datree/default.nix:10]:
```suggestion
    rev = version;
```

File: pkgs/development/tools/datree/default.nix [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+    sha256 = "0q399ydz6k3p88dlyk5qzki76wiiw3zvkfh40vh9y9wzala8dhhp";
+  };
+
+  vendorSha256 = "1l7k1ss0iqlkjldi0jqlpfrlzj1zkmbzi98m5p3d12rr84i7in2k";
+
+  ldflags = [
+    "-s"
```
Comment (SuperSandro2000) [inline pkgs/development/tools/datree/default.nix:17]:
```suggestion
    "-s" "-w"
```

**Comment by jceb** [comment]
@SuperSandro2000 thanks, I update the PR according to your requests.

**Pull request merged**